src/utils/hibp.js

```diff
@@ -175,14 +175,30 @@ function getAddressesAndLanguageForEmail (recipient) {
   }
 }
 
+/**
+ * Filter breaches that we would not like to show.
+ *
+ * @param {Array} breaches
+ * @returns {Array} filteredBreaches
+ */
+function getFilteredBreaches (breaches) {
+  return breaches.filter(breach => (
+    !breach.IsRetired &&
+    !breach.IsSpamList &&
+    !breach.IsFabricated &&
+    breach.IsVerified &&
+    breach.Domain !== ''
+  ))
+}
+
 /**
 A range of hashes can be searched by passing the hash prefix in a GET request:
 GET /breachedaccount/range/[hash prefix]
 
  * @param {string} sha1 first 6 chars of email sha1
- * @param {*} allBreaches
- * @param {*} includeSensitive
- * @param {*} filterBreaches
+ * @param {Array} allBreaches
```

Comment:
Nice, thanks for filling out type info. We should check that we have the eslint rule that checks this (it's not as good as the typescript compiler but still helpful)

Reply:
Following up in #2849.